README.md

```diff
@@ -65,9 +65,7 @@ Furthermore, since pydle is simply `asyncio`-based, you can run the client in yo
 import asyncio
 
 client = MyOwnBot('MyBot')
-loop = asyncio.get_event_loop()
```

Comment:
This appears to be incorrect. Changing the entry logic for my own implementation of pydle results in a hung program, which immediately terminates uncleanly. 

However, testing on Python 3.8 if I do not change my existing entry logic, the program starts properly. (Still stability-testing.)